lib/index.js

```diff
@@ -30,10 +31,17 @@ Outbound.prototype.initialize = function() {
   window.outbound.methods = [
     'identify',
     'track',
+    'alias',
     'registerApnsToken',
     'registerGcmToken',
     'disableApnsToken',
     'disableGcmToken',
+    'disableAllGcmTokens',
```

Comment:
@Tang8330 doesn't look to be an issue on our end (tests or ci) but we'll dig into it more. Most likely some changes you've made are causing issues on ie9 - so I'd start by testing your changes on ie9. You should be able to see the full suite of failures here https://circleci.com/gh/segment-integrations/analytics.js-integration-outbound/314